views/style.css

```diff
@@ -306,6 +307,8 @@ a.gameTile:hover {
   border: none;
   border-bottom: 1px solid #666666;
   color: white;
+  /* allow shrinking below min width */
+  width: 0;
```

Comment:
would `flex-shrink` work here? or `flex-basis` to combine `flex-grow` and `flex-shrink`

Reply:
No, browsers give `input` and `textarea` elements a default nonzero width (something like 100px). Flexbox can't force an element to shrink below it's width, so this is necessary.